server/graphql/v2/mutation/RootMutations.ts

```diff
@@ -251,4 +253,38 @@ export default {
       return moveExpenses(req, expenses, destinationAccount);
     },
   },
+  setTaxForm: {
+    type: new GraphQLObjectType({
+      name: 'SetTaxFormResult',
+      fields: {
+        success: { type: GraphQLBoolean },
+      },
+    }),
+    description: '[Root only] A mutation to set the tax from for an account.',
+    args: {
+      account: {
+        type: new GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to the Account the tax form should be set.',
+      },
+      taxFormLink: {
+        type: new GraphQLNonNull(GraphQLNonEmptyString),
```

Comment:
from `server/graphql/v2/scalar/URL.ts`

```suggestion
        type: new GraphQLNonNull(URL),
```